tutorials/phasorpy_multi-harmonic.py

```diff
@@ -0,0 +1,518 @@
+"""
+Multi-harmonic analysis with PhasorPy
+=====================================
+
+An introduction to manipulating multiple harmonics from fluorescence lifetime
+and hyperspectral images with the PhasorPy library.
+
+"""
+
+# %%
+# Import phasorpy
```

Comment:
I would prefer not to repeat the introduction tutorial verbatim (it will get out of sync sooner than later). Instead, provide a much condensed version following the introduction tutorial , and expand specifically on the multi-harmonic parts. Do you think that would still be readable? This tutorial would then belong in the API section.